gpii/configs/gpii.config.cloudBased.development.all.local.json

```diff
@@ -22,6 +22,6 @@
         "%preferencesServer/configs/gpii.preferencesServer.config.development.json",
         "%flatMatchMaker/configs/gpii.flatMatchMaker.config.development.json",
         "%rawPreferencesServer/configs/gpii.rawPreferencesServer.config.development.json",
-        "%deviceReporter/configs/gpii.deviceReporter.config.development.json"
+        "%deviceReporter/configs/gpii.deviceReporter.config.production.json"
```

Comment:
Yes, this is a change I made.  The reason being that this JIRA is about using the production version of the device reporter for acceptance testing.  I'm guessing from your comment that the proposed change is not specific to testing, or has nothing to do with testing at all.  Is that correct?